app/lib/actions/candlepin/consumer/bind_entitlement.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Candlepin
+    module Consumer
+      class BindEntitlement < Candlepin::Abstract
```

Comment:
In my related pr I've used `Actions::Katello::UpstreamSubscriptions::Delete` which might not be perfect naming-wise but I feel using the Candlepin::Consumer namespace isn't the best choice. Candlepin is an implementation detail of the upstream. From the sense of the API, and this Action, Candlepin is an irrelevant detail and that's why I went with the above namespace. Maybe Actions::UpstreamSubscriptions::* is better. Whaddya think?

Reply:
Funny, I was just typing out a similar disagreement to your namespacing. :)

> Candlepin is an implementation detail of the upstream

It's also an implementation detail of Katello, and we have some namespaces set up already for Candlepin-related actions. I'm not worried that we might one day bind entitlements to a consumer using something other than Candlepin. I'm also trying to limit the use of the name 'UpstreamSubscriptions' other than in reference to the API.

Reply:
I guess my argument is that this is more Candlepin specific than Katello specific, but :man_shrugging:, OK. I don't have strong feelings here.